src/utils.js

```diff
@@ -147,7 +162,10 @@ export async function* withPage(percy, url, callback, retry) {
 // Evaluate and return Storybook environment information from the about page
 /* istanbul ignore next: no instrumenting injected code */
 export function evalStorybookEnvironmentInfo({ waitForXPath }) {
-  return waitForXPath("//header[starts-with(text(), 'Storybook ')]", 5000)
+  let possibleEnvs = [];
```

Comment:
I think current tests are good enough.
We are already checking for `storybook/<number>`.

Current production is throwing `storybook/unknown` for storybook v7.

Reply:
I meant check for equality of this xpath in the test case and if it returns correct version.
or another option is when posting the snapshot we intercept the network call to verify the environment details being sent.

I its already there we can skip, this is to avoid regression when storybook is updated locally and its go untested.

Reply:
There is already test for this at -
https://github.com/percy/percy-storybook/blob/master/test/storybook.test.js#L245

Reply:
This is resolved in sync, there is already a test for this.